core/src/avm1.rs

```diff
@@ -187,7 +187,7 @@ impl<'gc> Avm1<'gc> {
     }
 
     /// Get the currently executing SWF version, if there is one.
-    fn current_swf_version(&self) -> Option<u8> {
+    pub fn current_swf_version(&self) -> Option<u8> {
         self.current_stack_frame().map(|sf| sf.swf_version())
```

Comment:
I'm actually adding a player version parameter in avm-fncleanup that gets passed to the action context. Perhaps this could take the action context and use that as default then?

Reply:
I think it's fine if it's in `Avm1` too, that way we don't have to pass the extra param. I think there will always be a stack frame anyway.

Reply:
@kmeisthax do you want me to make this change here, or leave it for you to make your changes?

Reply:
I merged in #84 so this is just returning a `u8` on master now